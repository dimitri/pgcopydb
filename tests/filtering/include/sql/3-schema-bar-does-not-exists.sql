  select nspname
    from pg_namespace
   where nspname !~ '^pg_' and nspname <> 'information_schema'
order by nspname;
