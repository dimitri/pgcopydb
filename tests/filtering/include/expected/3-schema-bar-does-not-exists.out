-[ RECORD 1 ]-----------------
nspname | app
-[ RECORD 2 ]-----------------
nspname | copy
-[ RECORD 3 ]-----------------
nspname | foo
-[ RECORD 4 ]-----------------
nspname | partitioned_tables
-[ RECORD 5 ]-----------------
nspname | public
-[ RECORD 6 ]-----------------
nspname | schema_name_20_chars
-[ RECORD 7 ]-----------------
nspname | seq

